client/src/main/java/org/apache/celeborn/client/ShuffleClientImpl.java

```diff
@@ -955,7 +955,9 @@ public int pushOrMergeData(
             @Override
             public void onSuccess(ByteBuffer response) {
               if (response.remaining() > 0 && response.get() == StatusCode.STAGE_ENDED.getValue()) {
```

Comment:
I think here should be MAP_ENDED

Reply:
@waitinfuture, I have addressed this comment but this doesn't make sense to me for the case that only Spark client upgrade without worker upgrade, which would process `STAGE_ENDED` status.